src/js/big-play-button.js

```diff
@@ -46,8 +47,16 @@ class BigPlayButton extends Button {
 
     // exit early if clicked via the mouse
     if (this.mouseused_ && event.clientX && event.clientY) {
+      const sourceIsEncrypted = this.player_.usingPlugin('eme') &&
+                          this.player_.eme.sessions &&
```

Comment:
should probably either line up or be indented once

Reply:
Yeah it was lined up in my editor but the indentation didn't carry over correctly for some reason. I'll figure it out.